R/run.R

```diff
@@ -191,6 +191,11 @@ log_image_to_run <- function(name, path = NULL, plot = NULL,
   if (is.null(run)) {
     run <- get_current_run()
   }
+  if (!is.null(plot)) {
```

Comment:
update the docstring for plot parameter?

Reply:
done